### ISSUE TITLE
Color problems

### ISSUE THREAD

**Issue by JESii** [vimoutliner/vimoutliner#174]
I had the same issue as originally reported #166 by @nilsbecker . To get around that and to use my standard `tabstop=2 shiftwidth=2`, I changed both values to 2 using `:setlocal`. That resolved the line coloring. However, overall, the coloring was unusable, with yellow values on a white background and the cursor line a dark grey which made that line unreadable.

![image](https://user-images.githubusercontent.com/486/103833622-49b53300-5036-11eb-9fe9-0036d259c573.png)

So I said fine -- I'll use the default color, which did improve the overall layout, but removed all coloring from indented lines.

![image](https://user-images.githubusercontent.com/486/103833694-78330e00-5036-11eb-9420-8d17e7fe5889.png)

I'm a developer, so I have customized a small number of my color settings for readability; any suggestions for how to make `vimoutliner` readable & color-friendly?
Thanks -- I'm looking forward to being able to use this for my text documents.

**Comment by JESii** [comment]
I found the mention of `votl_dark` and have been using that successfully, but it just stopped working and now I have nothing but a plain text file, even though it's named `fl.otl`. I'm in the GUI (MacVim), select that color scheme, but it doesn't color things at all.
Any suggestions as to what's going wrong?

**Comment by nilsbecker** [comment]
hi @JESii , so how did you get it to work with tabstop=2? `au FileType votl setlocal tabstop=2` and so on? is it working now?
(i kind of gave up and lived with incomplete coloring as the author of the colorscheme i'm using claims that vo does something dodgy and will not change the colorscheme, but if i change i might want to give it another try)

**Comment by JESii** [comment]
Yes, I did get it working, although every now and then it acts up.

I’ll get to my work computer computer tomorrow and let you know

Cheers..jon
Stay safe; be well.

Jon Seidel, CMC®
EDP Consulting, Inc. / www.edpci.com <http://www.edpci.com/>
 <http://www.4mypasswords.com/>Effective Decisions... Priceless!
Linked In Profile <http://www.linkedin.com/in/jonseidel>
Technology that means Business - Since 1979
mobile: 510-599-6130
 
Results that Make the Difference! The Certified Management Consultant (CMC) mark is awarded to consultants who meet the strict standards defined by the Institute of Management Consultants USA (an ISO/IEC 17024:2003 Certifying Body), including examination by their peers, client evaluations, and a written examination evidencing their understanding of the IMC USA's Code of Ethics. The CMC is the only consulting certification that has been ISO-accredited and recognized by the insurance industry as reducing risk.
Check out IMC USA <http://www.imcusa.org/?page=WHYHIREACMC> for more information.

**Comment by JESii** [comment]
@nilsbecker It's not just `tabstop=2`; it also requires `shiftwidth=2`. Plus I also have (already) set `softtabstop=2` as part of my normal setup. I found the default colorscheme (`votl_light` as I recall) to be quite unusable, so switched to `votl_dark` and that's working for me now.
I haven't setup an autogroup yet -- I just open a dedicated MacVim instance and use that strictly for `vimoutliner` files.
HTH
